### PR TITLE
Simplify web/blob build

### PR DESCRIPTION
**Pull request by grobie**
This merges `web/Makefile` and `web/blob/Makefile`, where the former's sole
purpose was to call the latter.

The the `blob-stamp` file can cause issues if it's present, but not the
`blob/files.go` file. As only the latter is the real dependency for the
build process, this indirection caused me some minutes to understand
why the build process was broken (don't ask me why I deleted files.go).
It can also be noted that all dependencies for `blob/files.go` are
outside of the `blob/` directory, so the separation isn't even logically
necessary.

The initial Makefile setup has been added in 3e97a36.

### CODE REVIEW

**Comment by brian-brazil** [comment]
:+1:

**Comment by beorn7** [comment]
Please merge with master first, the Makefile was already changed there.

If you remove web/Makefile, you have to change the top level Makefile, as it calls web/Makefile.

**Comment by grobie** [comment]
@beorn7 rebased. I missed indeed your recent change. I did not delete web/Makefile, so no changes anywhere else should be necessary.

**Comment by beorn7** [comment]
:+1:

File: web/Makefile [hunk 1]
```diff
@@ -11,12 +11,17 @@
...
-all: blob
+include ../Makefile.INCLUDE
 
-blob:
-	$(MAKE) -C blob
+all: blob/files.go
+
+SUFFIXES:
+
+# Do not embed map files and the non-minified bootstrap files.
```
Comment (juliusv) [inline web/Makefile]: How does the `find` below exclude those? Or was this meant as a TODO?

Comment (grobie) [inline web/Makefile]: I think this comment is meant just as a note that the embed-static.sh script won't include the files. @beorn7 might be able to explain this in more detail. This is the commit which introduced the line: https://github.com/prometheus/prometheus/commit/7e5745f

Comment (juliusv) [inline web/Makefile]: Ah, the exclusion happens in that script. All good then.

Comment (beorn7) [inline web/Makefile]: The note should actually explain it better. Like "embed-static.sh does not include ..."

**Pull request merged**